tern/analyze/docker/analyze.py

```diff
@@ -92,7 +92,7 @@ def execute_base_layer(base_layer, binary, shell):
     try:
         target = rootfs.mount_base_layer(base_layer.tar_file)
         rootfs.prep_rootfs(target)
-        common.add_base_packages(base_layer, binary, shell)
+        common.add_base_packages(base_layer, binary, shell, None)
```

Comment:
Instead of passing in `None` here, change the `add_base_packages function` definition to default `workdir` argument to a `None` value. i.e. `def add_base_packages(image_layer, binary, shell, work_dir=None)`